src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -43,7 +43,12 @@ export class ConnectedApps extends Component {
     const activeApps = apps ? apps.filter(app => !app.deleted) : [];
 
     const allAppsDeleted = deletedApps?.length === apps?.length;
-    const showHasNoConnectedApps = !apps || (allAppsDeleted && !loading);
+    // We treat this 404 'Record not found' error as an empty apps array
+    const firstError = errors?.[0];
```

Comment:
This is probably fine, but we tend to loop over the `errors` array and check to see if any entry matches the criteria we are looking for.

Reply:
I updated to use `some` instead 👍